fili-core/src/main/java/com/yahoo/bard/webservice/data/DruidQueryBuilder.java

```diff
@@ -202,10 +202,12 @@ protected GroupByQuery buildGroupByQuery(
                 intervals
         );
 
+        Filter mergedFilter = filter;
+
         // Override the grain with what's set in the template if it has one set
-        if (template.getTimeGrain() != null) {
-            granularity = template.getTimeGrain().buildZonedTimeGrain(timeZone);
-        }
+        Granularity mergedGranularity = template.getTimeGrain() != null ?
```

Comment:
I think I accidentally deleted one of your comment here @cdeszaq , ops sorry. Here is the original comment by rick.

`?` should go down on the next line when wrapping, as the `:` is down on the next line.

<sub>Applies to this same fix lower in the file as well</sub>